crates/client/rpc/src/errors.rs

```diff
@@ -36,6 +36,11 @@ pub enum StarknetRpcApiError {
     InternalServerError = 500,
     #[error("Failed to fetch pending transactions")]
     FailedToFetchPendingTransactions = 38,
+
+    // Provided hash is not a valid H256 hash.
+    // This may be reused in several places to check RPC params that are hashes.
+    #[error("Invalid hash")]
+    InvalidHash = 39,
```

Comment:
I propose a new `InvalidHash` that may be more generic for all future Hash related arguments. What do you think?